CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntax/SwiftSyntaxDoccIndex.swift

```diff
@@ -59,6 +59,8 @@ let nodesSections: String = {
       })
   )
 
+  addSection(heading: "Attributes", types: ATTRIBUTE_NODES.map(\.kind.syntaxType.description))
```

Comment:
Ah, I do have one comment after all. Could we sort the types alphabetically?

Reply:
Great idea! I've been also wondering about sorting the sections in alphabetical order. What do you think about that?

Reply:
I thought the other sections are alphabetically sorted, with the exception that the base node (e.g. `ExprSyntax`), its protocol type and the corresponding missing node are first, which makes sense to me.